Improve Contributing (#54)
meilisearch/firestore-meilisearch: CONTRIBUTING.md

⚠️ Before doing anything, make sure you got through the guide about [Releasing an Integration](https://github.com/meilisearch/integration-guides/blob/main/resources/integration-release.md).

1. Test the extension by installing it in a Firestore project:
```bash
firebase ext:install . --project=[your_project_id]
```

2. To update the version package in all needed files you can run the `version.sh` script with the version number in argument: `"X.X.X"`. If you want to modify it by hand you have to update the version in the following files:
- [`package.json`](/package.json).
- [`package.json`](/functions/package.json) in the functions directory.
- [`extension.yaml`](/extension.yaml).
- [`version.ts`](/functions/src/version.ts).

> After which, don't forget to rebuild the project to check if any changes has been forgotten on the `js` files in the `functions/lib` directory.

Reviewer: ```suggestion
After which, don't forget to rebuild the project to check if any changes have been overlooked on the `js` files in the `functions/lib` directory.
```

</br>
The `version.sh` script will update the version in all necessary files but you still need to check them:
```bash
sh script/version.sh X.X.X
```

3. Commit and push the modifications:

> ```

Reviewer 2: ```suggestion
```bash
```
